Formula/osm2pgrouting.rb

```diff
@@ -1,8 +1,10 @@
 class Osm2pgrouting < Formula
   desc "Import OSM data into pgRouting database"
   homepage "http://pgrouting.org/docs/tools/osm2pgrouting.html"
```

Comment:
@ilovezfs Would you mind bumping this URL to HTTPS? — To sync it with this (pending) PR: https://github.com/Homebrew/homebrew-core/pull/27394